tests/phpunit/testdata/plugins/test-plugin-unfiltered-uploads-with-errors/load.php

```diff
@@ -4,7 +4,7 @@
  * Plugin URI: https://github.com/WordPress/plugin-check
  * Description: Test plugin for the Localhost check.
  * Requires at least: 6.0
- * Requires PHP: 5.6
+ * Requires PHP: latest
```

Comment:
We have one test where we check for lack of error `plugin_header_invalid_requires_plugins`. This change has been done so that there is errors in `$errors` variable in the test and our targeted code is not there.